.azure-pipelines/ipu-tests.yml

```diff
@@ -81,5 +85,8 @@ jobs:
         source ${{ variables.poplar_sdk }}/poplar-ubuntu*/enable.sh
         source ${{ variables.poplar_sdk }}/popart-ubuntu*/enable.sh
 
-        python -m coverage run --source pytorch_lightning -m pytest pytorch_lightning tests -v --junitxml=$(Build.StagingDirectory)/test-results.xml --durations=50
+        # FIXME: remove -x
```

Comment:
can we remove the `-x`?

Reply:
Yep, but once most tests have been debugged. Probably don't merge this one before I remove the fixmes